core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -483,8 +483,9 @@ private[spark] class TaskSetManager(
           // a good proxy to task serialization time.
           // val timeTaken = clock.getTime() - startTime
           val taskName = s"task ${info.id} in stage ${taskSet.id}"
-          logInfo("Starting %s (TID %d, %s, %s, %d bytes)".format(
-              taskName, taskId, host, taskLocality, serializedTask.limit))
+          logInfo("Starting %s (TID %d, %s, %d, %s, %d bytes)".format(
```

Comment:
Others do that...I can't figure out the advantages and disadvantages.

There a lot sentence like:
logInfo("Finished task %s in stage %s (TID %d) in %d ms on %s (%d/%d)".format(
logError("Task %s in stage %s (TID %d) had a not serializable result: %s; not retrying"
            .format(i
 abort("Task %s in stage %s (TID %d) had a not serializable result: %s".format(


and also has this:
  logInfo(
            s"Lost task ${info.id} in stage ${taskSet.id} (TID $tid) on executor ${info.host}: " +
            s"${ef.className} (${ef.description}) [duplicate $dupCount]")

Need I to refactor it?